libs/common/src/types/mongo-db-models/tfm-deals.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectId } from 'mongodb';
+import { AuditDatabaseRecord } from '../audit-database-record';
+import { Deal } from './deal';
+
+export type TfmDealCancellation = {
+  reason: string;
+  bankRequestDate: number;
+  effectiveFrom: number;
+};
```

Comment:
Should this be stored in its own file a level up? We'll use the deal cancellation a lot not as a mongo-db-model

Reply:
Good idea, have moved